entrypoint

```diff
@@ -4,42 +4,66 @@ if [ -n "$DEBUG" ]; then
     set -x
 fi
 
-if ! find . -mindepth 1 | read -r; then
-    >&2 echo "Creating default configs..."
-    cp -r /opt/cfx-server-data/* /config
-    RCON_PASS="${RCON_PASSWORD-$(tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c 16)}"
-    sed -i "s/{RCON_PASS}/${RCON_PASS}/g" /config/server.cfg;
-    >&2 echo "----------------------------------------------"
-    >&2 echo "RCON password is set to: ${RCON_PASS}"
-    >&2 echo "----------------------------------------------"
-fi
+# Empty config args defaults to loading txAdmin
+CONFIG_ARGS=
 
-if [ -z "$NO_ONESYNC" ]; then
-    ONESYNC_ARGS="+set onesync on +set onesync_population true"
-fi
+# Temporary during the transition - this way old configs still trigger the old (no txadmin by default) behaviour.
+NO_TXADMIN = "DONTANSWERTHEPHONETOLESTERCREST"
 
-CONFIG_ARGS=
-if [ -z "${NO_DEFAULT_CONFIG}" ]; then
-    CONFIG_ARGS="$CONFIG_ARGS $ONESYNC_ARGS +exec /config/server.cfg"
+# Warn if NO_TXADMIN is the default.
+if [ "${NO_TXADMIN}" = "DONTANSWERTHEPHONETOLESTERCREST" ]; then
+    >&2 echo "------------------------------------------------------"
+    >&2 echo "WARNING - Default options will soon change to enabling"
+    >&2 echo "txAdmin by default. Set \$NO_TXADMIN to disable this."
```

Comment:
```suggestion
    >&2 echo "txAdmin by default. Set \$NO_TXADMIN to \`true\` to disable this."
```

Reply:
Doesn't need to be just true - any non-zero value works fine with our deprecation logic, and keeps consistency with the existing vars.